### ISSUE TITLE
React is not defined when using Trans component

### ISSUE THREAD

**Issue by Fabb111** [vinissimus/next-translate#821]
After upgrading from 1.0.7, when I use the Trans component, I get an "ReferenceError: React is not defined" error, originating from `next-translate/lib/cjs/formatElements.js (25:40)`.
This is caused by the Fragment transpiling to `React.createElement`, but React is not imported. Before this pull request was merged #641 (commit https://github.com/vinissimus/next-translate/commit/98149f0d90f2ef9dee063d2217ec4c49b9abdf63), React was imported in [src/Trans.tsx](https://github.com/vinissimus/next-translate/commit/98149f0d90f2ef9dee063d2217ec4c49b9abdf63#diff-33a06bf4b3df55337164dcbad98f248486fd9cb5c03eddf89e4437848c31491a)

```diff
- import React, {
-   cloneElement,
-   useMemo,
-   Fragment,
-   ReactElement,
-   ReactNode,
- } from 'react'
```

but it got removed in [src/formatElements.tsx](https://github.com/vinissimus/next-translate/commit/98149f0d90f2ef9dee063d2217ec4c49b9abdf63#diff-ac1569cb3eb66294b856f132172097db3463f6fd94d8d04946f70c69639f13fc)

```diff
+ import {
+   cloneElement,
+   Fragment,
+   ReactElement,
+   ReactNode,
+ } from 'react'
```

I wonder how this apparently isn't a problem for other users. Do I need to change something in my configs or is the best fix to just re-add React to the imports of `formatElements.tsx`?

**Comment by RobinSCU** [comment]
@Fabb111 I have the same problem; debugged it for an hour and now I'm happy that someone else has this issue too.

**Comment by GP-marco** [comment]
We are facing the same error as well but apparently only in one of our unit tests.
We didn't invested much time but worked around it by mocking this affected component.

**Comment by aralroca** [comment]
Thanks to report it! I fixed it, prereleased in [1.3.6-canary.4](https://github.com/vinissimus/next-translate/releases/tag/1.3.6-canary.4) 😊 If you can confirm that it works it would be great before releasing it. Thanks!
